.rubocop_todo.yml

```diff
@@ -1,21 +1,20 @@
 # This configuration was generated by
 # `rubocop --auto-gen-config`
-# on 2016-09-11 14:14:48 +0530 using RuboCop version 0.42.0.
+# on 2016-09-24 01:47:19 +0530 using RuboCop version 0.42.0.
```

Comment:
> Can't we ignore them?

No. Ignoring style guides of the repository is a slippery slope. You can increase the required the method length **once** if you think it default is too small. However, it should be done once and followed there after.